packages/bridge-ui/src/components/Toast.svelte

```diff
@@ -0,0 +1,66 @@
+<script lang="ts" context="module">
+  import { toast } from '@zerodevx/svelte-toast';
+
+  // Toast component exposes the following API:
+
+  export const errorToast = (msg: string) => {
+    toast.push(msg, {
+      theme: {
+        '--toastBackground': '#FF0000',
+        '--toastColor': '#e3e3e3',
+      },
+    });
+  };
+
+  export const successToast = (msg: string) => {
+    toast.push(msg, {
+      theme: {
+        '--toastBarBackground': '#fc0fc0',
+        '--toastBackground': '#4d4d4d',
+        '--toastColor': '#e3e3e3',
+      },
+    });
+  };
+</script>
+
+<script lang="ts">
+  import { SvelteToast, type SvelteToastOptions } from '@zerodevx/svelte-toast';
+
+  const toastOptions: SvelteToastOptions = {
+    dismissable: false,
+    duration: 4000,
+    pausable: false,
+  };
+
+  // TODO: props to control its possition:
+  //       top, bottom, left, right...
+</script>
+
+<div class="toast">
+  <SvelteToast options={toastOptions} />
+</div>
+
+<style>
+  /* TODO: Small devices. We need design!! */
+  .toast {
```

Comment:
![Screenshot 2023-03-22 at 4 27 22 PM](https://user-images.githubusercontent.com/108871478/226883294-83bdfcbc-7a60-46a3-8938-d466642918ba.png)
![Screenshot 2023-03-22 at 4 27 37 PM](https://user-images.githubusercontent.com/108871478/226883302-e42bbbfe-2c1a-4ed2-b72c-c5a0b25eaf35.png)


![Screenshot 2023-03-22 at 4 27 54 PM](https://user-images.githubusercontent.com/108871478/226883305-f1233af2-5cff-4027-810a-85999a4db4c1.png)

I tried the following values

```.toast {
    --toastContainerRight: 1rem;
    --toastContainerBottom: auto;
    --toastContainerTop: 5rem;
    --toastWidth: 100%;
    --toastMinHeight: 2rem;
    --toastPadding: 0 0.5rem;
    --toastBorderRadius: 0.4rem;
  }

@media (min-width: 640px) {
    .toast {
      /* --toastContainerRight: auto;
      --toastContainerLeft: calc(50vw - 8rem); */
      --toastWidth: 16rem;
    }
  }
```

What do you think about this?

Reply:
Changes done. Thanks